src/Components/Atoms/FATextField/index.tsx

```diff
@@ -1,49 +1,58 @@
-import React, { memo } from 'react';
-import { FormControl, TextField, InputAdornment, FormHelperText } from '@mui/material';
-import { IInput } from './Input.interface';
+import React, { memo } from "react";
+import {
+  FormControl,
+  TextField,
+  InputAdornment,
+  FormHelperText,
+} from "@mui/material";
+import { IInput } from "./Input.interface";
 
 export const FATextField = memo((props: IInput) => {
-  const {  disabled, variants, type, testID, value, label, adornment, adornmentPosition, helperText, onChange, ...materialUIProps } = props;
+  const {
+    disabled,
+    variants,
+    type,
+    testID,
+    value,
+    label,
+    adornment,
+    adornmentPosition,
+    helperText,
+    onChange,
+    ...materialUIProps
+  } = props;
 
   const AddAdornment = () => {
-    return (
-      <InputAdornment position="start">
-        {adornment}
-      </InputAdornment>
-    );
+    return <InputAdornment position="start">{adornment}</InputAdornment>;
   };
 
-  const startAdornment = adornment && adornmentPosition === 'start' ? (
-    AddAdornment()
-  ) : null;
+  const startAdornment =
+    adornment && adornmentPosition === "start" ? AddAdornment() : null;
 
-  const endAdornment = adornment && adornmentPosition === 'end' ? (
-    AddAdornment()
-  ) : null;
+  const endAdornment =
+    adornment && adornmentPosition === "end" ? AddAdornment() : null;
 
   return (
     <FormControl data-testid={testID}>
```

Comment:
`<FormControl>` and the helper text should have their own component in molecules. So, exclude it from this component.

Reply:
noted